batch/requirements.txt

```diff
@@ -8,4 +8,4 @@ plotly>=5.18.0,<6
 # Worker requirements
 aiodocker>=0.17.0,<1
 aiorwlock>=1.0.0,<2
-async-timeout>=4.0.2,<5
+async-timeout>=4.0.2,<6
```

Comment:
This was needed to let aiohttp update. The only usages were in worker.py ([eg](https://github.com/hail-is/hail/blob/main/batch/batch/worker/worker.py#L1012)) which seems totally safe with version 5